examples/framework/preferences/minimalEditor/js/minEditor.js

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one of these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+/* global fluid */
+
+(function () {
+    "use strict";
+
+    /**
+     * Panel for the auto-pilot preference
+     */
+    fluid.defaults("minEditor.panels.autoPilot", {
+        gradeNames: ["fluid.prefs.panel"],
+
+        // the Preference Map maps the information in the primary schema to this panel
+        preferenceMap: {
+            // the key must match the name of the pref in the primary schema
+            "minEditor.autoPilot": {
+                // this key is the path into the panel's model where this preference is stored
+                "model.autoPilot": "default"
+            }
+        },
+
+        // selectors identify elements in the DOM that need to be accessed by the code;
+        // in this case, the Renderer will render data into these particular elements
+        selectors: {
+            autoPilot: ".mec-autoPilot"
+        },
+
+        // the ProtoTree is basically instructions to the Renderer
+        // the keys in the prototree match the selectors above
+        protoTree: {
+            // this value is a reference to the last part of the model path in the preferenceMap
+            autoPilot: "${autoPilot}"
+        }
+    });
+
+    /**
+     * Initialize and instantiate the editor
+     */
+    minEditor.init = function (container) {
```

Comment:
Spoke with @amb26 in the [channel](https://botbot.me/freenode/fluid-work/2015-11-12/?msg=53997194&page=2) about this today. In general this isn't the best approach but currently the fluid..prefs.create function doesn't provide much support for returning a reusable grade (see: https://issues.fluidproject.org/browse/FLUID-5817). What that means for now is to keep using the current implementation, but leave a TODO referencing the FLUID-5817.